mlcomp/parallelm/mlpiper/main.py

```diff
@@ -127,6 +130,9 @@ def _add_deps_sub_parser(subparsers):
     group.add_argument('-f', '--file', type=argparse.FileType('r'),
                        help='A json file path, whose content is a pipeline. Or component JSON')
 
+    deps.add_argument('-r', '--comp-root', default=None, required=True,
+                                help='MLPiper components root dir. Recursively detecting components')
+
```

Comment:
I wanted you to add a new option which will only show the dependancies not install them.,